nncf/pruning/export_helpers.py

```diff
@@ -345,6 +345,23 @@ def mask_propagation(self, model: NNCFNetwork, nx_node, graph: NNCFGraph, nx_gra
             assert all([torch.allclose(input_masks[0], mask) for mask in input_masks])
         nx_node['output_mask'] = input_masks[0]
 
+    def input_prune(self, model: NNCFNetwork, nx_node: dict, graph: NNCFGraph, nx_graph: nx.DiGraph):
```

Comment:
This is not a proper way to register a user module as having trainable parameters to compress.

Use the `@nncf.register_module` decorator on a target module, such as `L2Norm`.

Reply:
@AlexanderDokuchaev could you successfully use the proposed way? Is the PR unnecessary now, or are investigations still underway?

Reply:
Updated with using decorator @register_module